kbi/0032/index.rst

```diff
@@ -0,0 +1,58 @@
+.. index::
+   single: RIA; cross-platform
+   single: datalad-next; RIA
+
+KBI0032: Pushing to RIA-store between OSs, and other improvements
+=================================================================
+
+:authors: Michał Szczepanik <m.szczepanik@fz-juelich.de>
+:discussion: https://github.com/psychoinformatics-de/knowledge-base/pull/125
+:keywords: datalad-next, RIA, macOS, linux, windows, cross-platform
+:software-versions: datalad-next_1.4.1, datalad_1.0.2
+
+DataLad versions prior to at least 1.0.2 [#f1]_ can display the
```

Comment:
I think it is still very worthwhile to have this KBI, but given that the linked PR is merged, I would propose to remove the footnote, and edit the version number:
```suggestion
DataLad versions prior to 1.1.2 can display the
```